release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3360,6 +3360,43 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.43
+[id="ocp-4-16-43_{context}"]
+=== RHSA-2025:9765 - {product-title} {product-version}.43 bug fix and security update
+
+Issued: 02 July 2025
+
+{product-title} release {product-version}.43 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:9765[RHSA-2025:9765] advisory. 
```

Comment:
So is this a 4.16+ thing where we are using {product-version} instead of hard coding?